engine/apps/schedules/ical_utils.py

```diff
@@ -61,7 +60,6 @@
 def users_in_ical(
     usernames_from_ical: typing.List[str],
     organization: "Organization",
-    users_to_filter: typing.Optional["UserQuerySet"] = None,
 ) -> typing.Sequence["User"]:
```

Comment:
we can probably change the return type here. I believe it was like this to accommodate the union between a list and a queryset